integration_tests/integration/createLicence.spec.ts

```diff
@@ -53,6 +54,11 @@ context('Create a licence', () => {
       .enterAddress()
       .nextInput()
       .checkBoxes()
+      .nextInput(false) // aria-expanded attribute causes issues with Axe
+      .enterTime('10', '30', 'curfewStart')
```

Comment:
This might be a legitimate accessibility issue. But one for the design guys to worry about.